### PR TITLE
Feature/adminpage

### PR DESCRIPTION
**Pull request by AndrZakhara**

### CODE REVIEW

File: src/db.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      rate: 1.11,
+      amountFrom: 100,
+      amountTo: 111,
+    },
+  ],
+};
+
+export { users, data };
+
+// this.listOfUsers = [
```
Comment (ZelikSV) [inline src/db.js]: You must put this in mocks

File: src/components/Admin/UserList.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
```
Comment (ZelikSV) [inline src/components/Admin/UserList.js]: make this import construction shortly with destructuring

File: db.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    role: 'user',
+    ava: [], // массив байт
+  },
+  'vasya@gmail.com': {
+    firstName: 'Vasya',
+    lastName: 'Poopkin',
+    phone: '380672386289',
+    email: 'vasya@gmail.com',
+    role: 'user',
+    ava: [], // массив байт
```
Comment (Bodyan322) [inline db.js]: u need to remove comments

File: src/components/Admin/UserList.js (hunk 2 above)
Comment (AlexYalovoy) [reply src/components/Admin/UserList.js]: Do you mean something like this?
`import {List} from '@material-ui/core'`

Comment (ZelikSV) [reply src/components/Admin/UserList.js]: Yes it is

File: db.js (hunk 3 above)
Comment (AndrZakhara) [reply db.js]: ok

File: src/components/Admin/UserList.js (hunk 2 above)
Comment (AndrZakhara) [reply src/components/Admin/UserList.js]: fixed

File: db.js (hunk 3 above)
Comment (AndrZakhara) [reply db.js]: I can't find it in the project ))

**Pull request merged**